src/test/java/org/folio/rest/api/PatronActionSessionAPITest.java

```diff
@@ -173,18 +180,126 @@ public void canUpdatePatronActiveSession() throws InterruptedException, Malforme
   public void cannotUpdateNonExistentActionSession() throws InterruptedException, MalformedURLException,
     TimeoutException, ExecutionException {
 
-    JsonObject nonExistPatronActionSession = createSession("Check-out");
+    JsonObject nonExistPatronActionSession = createPatronActionSession("Check-out");
     JsonResponse response = assertRecordClient.attemptPutById(nonExistPatronActionSession);
 
     assertThat(response.getStatusCode(), is(404));
   }
 
-  private JsonObject createSession(String actionType){
+  @Test
+  public void canGetPatronActionSessionStorageExpiredSessionPatronIds() throws InterruptedException,
+    ExecutionException, TimeoutException, MalformedURLException {
+
+    String firstPatronId = UUID.randomUUID().toString();
+    createPatronActionSessionRecords(firstPatronId, "Check-in", DateTime.now().minusDays(3));
+    createPatronActionSessionRecords(firstPatronId, "Check-out", DateTime.now().minusDays(3));
+    String secondPatronId = UUID.randomUUID().toString();
+    createPatronActionSessionRecords(secondPatronId, "Check-in", DateTime.now().minus(1));
+    createPatronActionSessionRecords(secondPatronId, "Check-out", DateTime.now().minus(1));
+
+    assertThat(getExpiredSessionPatronIds("Check-out", 10, DateTime.now()).size(), is(2));
+
+    JsonArray checkInJsonArray = getExpiredSessionPatronIds("Check-in", 10, DateTime.now().minusDays(2));
+    assertThat(checkInJsonArray.size(), is(1));
+    assertThat(checkInJsonArray.getJsonObject(0).getString("patronId"), is(firstPatronId));
+    assertThat(checkInJsonArray.getJsonObject(0).getString("actionType"), is("Check-in"));
+
+    JsonArray checkOutJsonArray = getExpiredSessionPatronIds("Check-out", 10, DateTime.now().minusDays(2));
```

Comment:
Could you add test case to get expired sessions without and incorrect action type?